### PR TITLE
Oath app redirect insensitive to trailing slash

### PR DESCRIPTION
**Pull request by chriddyp**
cc @cldougl

### CODE REVIEW

File: dash_auth/plotly_auth.py [hunk 1]
```diff
@@ -244,7 +244,7 @@ def create_or_overwrite_dash_app(filename, sharing, app_url):
 
 
 def create_or_overwrite_oauth_app(app_url, name):
-    redirect_uri = '{}_oauth-redirect'.format(app_url)
+    redirect_uri = '{}/_oauth-redirect'.format(app_url.strip('/'))
```
Comment (cldougl) [inline dash_auth/plotly_auth.py]: 🎉

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,18 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [0.0.10] - 2017-10-05
+### Fixed
+- The oauth redirect URL is no trailing-backslash insensitive
```
Comment (cldougl) [inline CHANGELOG.md]: I think this is a tiny bit hard to parse with the double negative
it's no longer sensitive to a trailing-backslash right?

Comment (chriddyp) [reply CHANGELOG.md]: woops left out the "longer"
